.github/workflows/build.yml

```diff
@@ -33,18 +33,22 @@ jobs:
       - name: Build the extension
         run: |
           set -eux
+          export LANG=en_US.UTF-8
+          export LANGUAGE=en_US:en
+          export LC_ALL=en_US.UTF-8
```

Comment:
What are these environment variables used for?

Reply:
The `browser_check` test of JupyterLab has failed because of this error https://github.com/QuantStack/jupyter-suggestions/actions/runs/12285648889/job/34284127425#step:7:395

I cannot reproduce it on my machine, a quick Google search led to these variables.

Reply:
logged into #14